tests/full_matrix/full_matrix_move.cc

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
Can you please add the standard preamble, including a one- or two-line comment that says what the test checks?